core/tests/test_utils.h

```diff
@@ -0,0 +1,290 @@
+// -------------------------------------------------------------
+//  Cubzh Core Unit Tests
+//  test_utils.h
+//  Created by Nino PLANE on October 10, 2022.
+// -------------------------------------------------------------
+
+#pragma once
+
+#include "utils.h"
+#include "config.h"
+
+void test_utils_float_isEqual(void) {
+
+    float f1 = 1.0f;
+    float f2 = 2.0f;
+
+    TEST_CHECK(float_isEqual(1.0f, 1.0f, 0.5f));
+    TEST_CHECK(float_isEqual(10000.0f, 10000.0f, 0.5f));
+    TEST_CHECK(float_isEqual(f1, f2, 0.5f) == false);
```

Comment:
You don't really need to use variables `f1` and `f2` here I think.
You can use literals `1.0f` and `2.0f`, for a better readability.